test/e2e/test.js

```diff
@@ -0,0 +1,231 @@
+'use strict';
+/* jshint ignore:start */
+const {expect} = require('chai');
+const startInsightUI = require('../../lib/test/startInsightUI');
+const wait = require('../../lib/test/util/wait');
+
+const topPanel = require('../../lib/test/pages/TopPanel');
+const blockPage = require('../../lib/test/pages/BlockPage');
+const statusPage = require('../../lib/test/pages/StatusPage');
+
+const InsightUIOptions = require('../../lib/test/service-ctl/InsightUIOptions');
+
+let originalTimeout;
+
+describe('basic UI tests', () => {
+    let masterNode;
+
+    let url;
+    let blockHash;
+    let trxHash;
+
+    beforeAll(async () => {
+        const rootPath = process.cwd();
+
+        const insightUIContainerOptions = {
+            volumes: [
+                `${rootPath}/dashcore-node:/insight/node_modules/@dashevo/insight-ui/dashcore-node`,
+                `${rootPath}/po:/insight/node_modules/@dashevo/insight-ui/po`,
+                `${rootPath}/public:/insight/node_modules/@dashevo/insight-ui/public`,
+            ],
+        };
+
+        const insighUIOptions = {
+            cacheNodeModules: true,
+            localAppPath: rootPath,
+            container: insightUIContainerOptions,
+        };
+
+        InsightUIOptions.setDefaultCustomOptions(insighUIOptions);
+
+
+        [masterNode] = await startInsightUI.many(1);
+
+        url = `http://127.0.0.1:${masterNode.insightUi.options.getUiPort()}/insight/`;
+
+        await masterNode.dashCore.getApi().generate(15);
+
+    });
+
+    afterAll(async () => {
+        const instances = [
+            masterNode,
+        ];
+
+        await Promise.all(instances.filter(i => i)
+            .map(i => i.remove()));
+
+    });
+
+    beforeEach(function () {
+        originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = 250000;
+    });
+
+    afterEach(function () {
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout;
+    });
+
+    describe('Home Page', () => {
+        it('should be able to open main page', async () => {
+            await browser.get(url);
+            const title = await browser.getTitle();
+            expect(title).equal('Home | Insight');
+        });
+
+        it('should be able to open block page', async () => {
+            await topPanel.openBlockPage();
+            const title = await browser.getTitle();
+            expect(title).equal('Home | Insight');
+        });
+
+        it('should be able to open status page', async () => {
+            await topPanel.openStatusPage();
+            await wait(10000); // time to complete sync
+            const title = await browser.getTitle();
+            expect(title).equal('Status | Insight');
```

Comment:
expect.to.equal

Reply:
expect.to.equal doesn't work here
it uses jasmine's modules

Message:
    Failed: Cannot read property 'equal' of undefined
  Stack:
    TypeError: Cannot read property 'equal' of undefined
        at UserContext.it (/Users/andrei/dash_src/insight-ui/test/e2e/test.js:87:23)
        at process._tickCallback (internal/process/next_tick.js:68:7)
    From: Task: Run it("should be able to open status page") in control flow
        at UserContext.<anonymous> (/Users/andrei/dash_src/insight-ui/node_modules/jasminewd2/index.js:94:19)
        at /Users/andrei/dash_src/insight-ui/node_modules/jasminewd2/index.js:64:48

Reply:
Oh, my bad. I thought it’s chai.